src/test/regress/greengage_schedule

```diff
@@ -119,8 +119,7 @@ test: dtm_retry
 
 # The appendonly test cannot be run concurrently with tests that have
 # serializable transactions (may conflict with AO vacuum operations).
-test: rangefuncs_cdb gp_dqa subselect_gp subselect_gp2 gp_transactions olap_group olap_window_seq sirv_functions appendonly create_table_distpol alter_distpol_dropped query_finish partial_table subselect_gp_indexes
-
+test: rangefuncs_cdb gp_dqa subselect_gp subselect_gp2 gp_transactions olap_group olap_window_seq sirv_functions appendonly create_table_distpol alter_distpol_dropped query_finish partial_table subselect_gp_indexes matfuncscan
```

Comment:
The `matfuncscan` test runs in parallel with many other tests. It's important to ensure that tables and functions with the same names aren't used to avoid conflicts. It's better to use longer names specific to the test, or make the tables temporary.

Reply:
or you may use setting schema in beginning of sql file and dropping it in the end, like in `src/test/regress/sql/bfv_joins.sql`